include/range/v3/view/stride.hpp

```diff
@@ -244,20 +255,34 @@ namespace ranges
             {
                 return adaptor<false>{*this};
             }
+#ifdef RANGES_WORKAROUND_MSVC_711347
+            CONCEPT_REQUIRES(const_iterable)
+#else // ^^^ workaround / no workaround vvv
             CONCEPT_REQUIRES(const_iterable())
+#endif // RANGES_WORKAROUND_MSVC_711347
```

Comment:
These edits are unfortunate.

Reply:
Is there a different style you'd prefer? I didn't give this much thought under the expectation that I'll be able to use the variable-template workaround for the "normal" case on the new-concepts-syntax branch.

Reply:
I was merely complaining about a large number of workarounds, the only purpose of which was the addition or removal of `()`. You're right, this should go away in the new-concepts-syntax branch.